classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+# class  trimmer decorator
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0...10].to_s
```

Comment:
- [OPTIONAL] You can also test If your name is long and has a maximum of 10 before you Trimmer it by adding a test length condition like this 👇🏻 

```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```